terraform/modules/services/fat-buyer-ui/ecs.tf

```diff
@@ -141,6 +141,10 @@ resource "aws_ecs_task_definition" "fat_buyer_ui" {
             {
                 "name": "GUIDED_MATCH_SERVICE_API_KEY",
                 "valueFrom": "${var.fat_api_key_arn}"
+            },
+            {
+                "name": "ROLLBAR_ACCESS_TOKEN",
+                "valueFrom": "${var.rollbar_access_token}"
```

Comment:
This will need to be the ARN of the secure SSM param - not the value

Reply:
but on this [file](https://github.com/Crown-Commercial-Service/ccs-scale-infra-services-fat/blob/a8991d0057c123fb4706ed6cfa25e395070d5e2d/terraform/modules/configs/deploy-all/main.tf#L242), the name of the variable is `rollbar_access_token`